plugin.video.earthcam/resources/lib/earthcam.py

```diff
@@ -0,0 +1,230 @@
+#   Copyright (C) 2018 Lunatixz
+#
+#
+# This file is part of EarthCam.
+#
+# EarthCam is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# EarthCam is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EarthCam.  If not, see <http://www.gnu.org/licenses/>.
+
+# -*- coding: utf-8 -*-
+import sys, time, datetime, re, traceback
+import urlparse, urllib, urllib2, socket, json
+import xbmc, xbmcgui, xbmcplugin, xbmcaddon
+
+from bs4 import BeautifulSoup
+from simplecache import SimpleCache, use_cache
+
+# Plugin Info
+ADDON_ID      = 'plugin.video.earthcam'
+REAL_SETTINGS = xbmcaddon.Addon(id=ADDON_ID)
+ADDON_NAME    = REAL_SETTINGS.getAddonInfo('name')
+SETTINGS_LOC  = REAL_SETTINGS.getAddonInfo('profile')
+ADDON_PATH    = REAL_SETTINGS.getAddonInfo('path').decode('utf-8')
+ADDON_VERSION = REAL_SETTINGS.getAddonInfo('version')
+ICON          = REAL_SETTINGS.getAddonInfo('icon')
+FANART        = REAL_SETTINGS.getAddonInfo('fanart')
+LANGUAGE      = REAL_SETTINGS.getLocalizedString
+
+## GLOBALS ##
+TIMEOUT       = 15
+CONTENT_TYPE  = 'files'
+DEBUG         = REAL_SETTINGS.getSetting('Enable_Debugging') == 'true'
+BASE_URL      = 'http://www.earthcam.com'
+NET_URL       = '%s/network/'%BASE_URL
+LOGO_URL      = 'http://icons.better-idea.org/icon?url=%s&size=70..120..200'
+MAIN_MENU     = [("Browse Featured", NET_URL, 2),
```

Comment:
Can you not hardcode those strings?

Reply:
NET_URL? I could... Style preference I guess..

Reply:
I mean the main menu items

Reply:
Never really thought about it... I usually design the Main menu as a tuple... Force of habit =)

Reply:
What I'm trying to say is not to hardcode language dependent strings such those of the main menu. Please move them to a strings.po file. Others may want to provide translations for them

Reply:
Yes, you're right... probably best to move to language string.